feat: 66812 automatic task assignment (#372)
valtimo-platform/valtimo-documentation: using-valtimo/document/automatic-task-assignment.md

Reviewer: Please also add this file to `SUMMARY.md`.

# Automatic task assignment

> It is possible to have user tasks be automatically assigned to a Case Assignee making the tasks in a Case appear under 

Reviewer: ```suggestion
It is possible to have user tasks be automatically assigned to a case assignee making the tasks in a case appear under 
```

> the Case Assignee's Task List.

Reviewer: ```suggestion
the case assignee's task list.
```

## Functionality

> This feature causes User Tasks (existing and new) to be assigned to the current case assignee if there is one assigned.  

Reviewer: ```suggestion
This feature causes user tasks (existing and new) to be assigned to the current case assignee if there is one assigned.  
```

> Upon the assignment or change of a user to a case all user tasks from all processes currently using the same business key will 

Reviewer 2: I think the first part of this sentence is not that easy to red. Maybe change to '(re-)assignment of a case'

have their assignee updated to match the new case assignee.  
Removal of an assignee from the case will also clear the assignee on all assigned user tasks withing all processes 
using the same business key.

## Prerequisites

This feature requires the following:
- Valtimo 10.7.0 or newer

## How to configure automatic assignment of tasks

> Automatic assignment of tasks can be enabled at the bottom of a Case Configuration page (`Admin` -> `Case` -> 

Reviewer: ```suggestion
Automatic assignment of tasks can be enabled at the bottom of an individual case configuration page (`Admin` -> `Case` -> 
```

`[your case definition]`). This option becomes visible when the Case is allowed to have an assignee.

![Configuring automatic task assignment](img/case-automatic-task-assignment.png)

## Limitations

> * Only tasks that the user has access to (User has the required role) are assigned to the case assignee

Reviewer: ```suggestion
* Only tasks that the user has access to (user has the required role) are assigned to the case assignee
```
